docs/templates/redmine_template.txt.j2

```diff
@@ -1,5 +1,16 @@
 h1. EC-Earth 4: Monitoring
```

Comment:
Unrelated to this PR, but I would get rid of this headline. Within the Redmine context, it is pretty clear that it is about monitoring of an EC-Earth experiment. The issue headline serves as a better focus for the reader.